### PR TITLE
ApiKey

### PR DESCRIPTION
**Pull request by EmmaluFox**
-Added Nasa api key to .env and added .env to gitignore

### CODE REVIEW

**Comment by MichaelWalker** [comment]
Nice use of `.env` files! :) 

Couple of questions though

1) How will we set these keys once they are deployed?
2) How can we let the other developers (or future devs) know what the need to do?

Also, by convention, environment variables tend to have UPPERCASE_NAMES, so I'd probably expect this one to be `REACT_APP_NASA_API_KEY`

**Review by MichaelWalker — CHANGES_REQUESTED** [review]
Nice use of .env files! :)

Couple of questions though

1) How will we set these keys once they are deployed?
2) How can we let the other developers (or future devs) know what the need to do?

Also, by convention, environment variables tend to have UPPERCASE_NAMES, so I'd probably expect this one to be `REACT_APP_NASA_API_KEY`

File: README.md [hunk 1]
```diff
@@ -1,44 +1,4 @@
...
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+Accessing API Key:
+ You can access the NASA api key with: process.env.REACT_APP_NASA_API_KEY
```
Comment (MichaelWalker) [inline README.md:5]: The most important bits of information here is what the dev needs to do before the app will work for them!

Think of it as an easy to follow guide for getting a new developer up to speed with the project.

The `process.env` stuff we can probably assume a developer has seen before, as its just a standard feature of node.

Could we have something like
````Markdown
# Mars Mission

An educational website for inspiring interest in Mars.

You can view it at https://techswitch-mars-mission.herokuapp.com/
(or our staging environment at https://techswitch-mars-mission-stage.herokuapp.com/)

## Setup
The app is a react application built using Create React App.

You should be able to start it using 
`npm start`

And run the tests using
`npm test`

### Environment Variables
We use `.env` files to manage our environment variables. This should live in the root of your application.
The environment variables used are:

*REACT_APP_NASA_API_KEY:* Your API for the NASA API (obtained from https://api.nasa.gov/)
```

**Review by MichaelWalker — CHANGES_REQUESTED** [review]

**Review by EmmaluFox — COMMENTED** [review]
Added instructions to README

**Review by EmmaluFox — COMMENTED** [review]
Accept

**Review by EmmaluFox — COMMENTED** [review]
.

**Review by MichaelWalker — APPROVED** [review]

**Pull request merged**